### PR TITLE
Devices - add device dialog and paper key nudge

### PR DESCRIPTION
**Pull request by buoyad**
Adds a popup after clicking 'Add a device' on the devices tab.
add-device-demo-desktop
add-device-demo-mobile

Also reroutes the paper key todo button to this dialog. r? @keybase/react-hackers cc @keybase/design

### CODE REVIEW

**Comment by cecileboucheron** [comment]
@buoyad Icons are a bit too big I think, we usually do 48 on desktop and 64 on mobile. And how about the buttons say 'Add a computer', 'Add a phone', 'Create a paper key'? Just to remind people what will happen on click.

**Comment by cecileboucheron** [comment]
@buoyad Also our hover background is usually blue_4 (let's not use the drop shadow for hovers)

**Comment by buoyad** [comment]
@keybase/react-hackers bump

File: shared/devices/add-device/index.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    </Kb.Box2>
+  </Kb.ScrollView>
+)
+
+const DeviceBox = Styles.isMobile
+  ? Kb.Box2
+  : Styles.styled(Kb.Box2)({
+      ...Styles.transition('background-color'),
+      '&:hover': {
+        backgroundColor: Styles.globalColors.blue4,
```
Comment (chrisnojima) [inline shared/devices/add-device/index.js]: can just use css for this right, we generate the background hover css automatically

File: shared/devices/add-device/index.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      },
+      border: `1px solid ${Styles.globalColors.black_05}`,
+      borderRadius: Styles.borderRadius,
+      padding: Styles.globalMargins.tiny,
+      width: 140,
+    })
+const bigIcon = isLargeScreen && Styles.isMobile
+const typeToIcon = {
+  computer: bigIcon ? `icon-computer-96` : `icon-computer-64`,
```
Comment (chrisnojima) [inline shared/devices/add-device/index.js]: surprised prettier let you use backticks here

File: shared/devices/index.js [hunk 3]
```diff
@@ -164,7 +142,4 @@ const revokedHeaderStyles = Styles.styleSheetCreate({
   },
 })
 
-export default compose(
-  Kb.OverlayParentHOC,
-  Kb.HeaderOnMobile
-)(Devices)
+export default compose(Kb.HeaderOnMobile)(Devices)
```
Comment (chrisnojima) [inline shared/devices/index.js]: no need for compose anymore

File: shared/devices/add-device/index.js (hunk 1 above)
Comment (buoyad) [reply shared/devices/add-device/index.js]: I made the other changes in https://github.com/keybase/client/pull/15972/commits/4162e5de421fbe3afa86547860709fc564f1d85e, I can't find the bg hover css generation

Comment (buoyad) [reply shared/devices/add-device/index.js]: https://github.com/keybase/client/pull/15972/commits/388ca0ebeca57598a7433b714c2561735b40f95b uses className instead

**Comment by buoyad** [comment]
@chrisnojima addressed feedback, PTAL

**Comment by buoyad** [comment]
- [ ] Cherry-pick fixes from #15972 before merge

**Pull request merged**